insanely_fast_whisper_rocm/webui/ui.py

```
@@ -187,14 +198,26 @@ def create_ui_components(
-    """Create and return Gradio UI components with all parameters."""
+    """
+    Builds and returns the Gradio Blocks app for the Insanely Fast Whisper web UI.
+    
+    Creates a complete Gradio interface including file upload, model and processing configuration controls, timestamp stabilization options, task and file-handling controls, outputs (transcription text and raw JSON), download buttons, and the submit action wired to the transcription handler.
+    
+    Parameters:
+        default_model (str): Default model name shown in the model configuration textbox.
+        default_stabilize (bool): Default state for the word-level timestamp stabilization checkbox.
+        default_demucs (bool): Default state for the Demucs noise reduction checkbox.
+        default_vad (bool): Default state for the voice activity detection (VAD) checkbox.
+        default_vad_threshold (float): Default threshold value for VAD (range 0.1–0.9).
```

Ruff (0.14.7)

Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)